disco/extensions/upgrade_simulation/upgrades/common_functions.py

```diff
@@ -399,12 +650,79 @@ def get_all_line_info(compute_loading=True, upper_limit=1.5, ignore_switch=True)
             else:
                 all_df.at[index, "status"] = "normal"
     all_df = all_df.reset_index(drop=True).set_index('name')
+    all_df["kV"] = all_df["kV"].round(5)
     # if switch is to be ignored
     if ignore_switch:
         all_df = all_df.loc[all_df['Switch'] == False]
     return all_df.reset_index()
 
 
+def compare_multiple_dataframes(comparison_dict, deciding_column_name, comparison_type="max"):
+            """This function compares all dataframes in a given dictionary based on a deciding column name
+
+            Returns
+            -------
+            Dataframe
+            """
+            summary_df = pd.DataFrame()
+            for df_name in comparison_dict.keys():
+                summary_df[df_name] = comparison_dict[df_name][deciding_column_name]
+            if comparison_type == "max":
+                label_df = summary_df.idxmax(axis=1)  # find dataframe name that has max 
+            elif comparison_type == "min":
+                label_df = summary_df.idxmax(axis=1)  # find dataframe name that has max 

```

Comment:
typo? `# find dataframe name that has min`

Reply:
Oh yes, changed.